backend/s3/s3.go

```diff
@@ -4346,7 +4346,7 @@ func (f *Fs) copy(ctx context.Context, req *s3.CopyObjectInput, dstBucket, dstPa
 		req.StorageClass = &f.opt.StorageClass
 	}
 
-	if src.bytes >= int64(f.opt.CopyCutoff) {
+	if src.bytes >= int64(f.opt.CopyCutoff) && f.opt.CopyCutoff > 0 {
```

Comment:
This works fine unless the size of the object is > 5 GB then the single part copy will fail.

I think we should probably insert a check here instead and return an error to the user when initialising the Fs

https://github.com/rclone/rclone/blob/52d444f4a91cc7afd1552574dced484fd3b5138b/backend/s3/s3.go#L3252

Reply:
Thanks for reminding that. I didn't aware of single part copy before..
Now I refactor it to verify when initialing the backend.
please take a look